bin/sign-zip-upload.sh

```diff
@@ -19,17 +19,25 @@ command -v jq >/dev/null 2>&1 || {
 if [[ -z "${1}" ]]; then
   echo "Missing folder name in current working directory"
   echo "Folder contents will be considered up to standards, zipped and uploaded to blob storage"
-  echo "\n  Re-run script \`./sign-zip-upload.sh <folder-name>\`\n"
+  echo "\n  Re-run script \`./sign-zip-upload.sh <folder-name> <zip-file-name>\`\n"
+
+  exit 1
+fi
+
+if [[ -z "${2}" ]]; then
+  echo "Missing zip file name"
+  echo "\n  Re-run script \`./sign-zip-upload.sh <folder-name> <zip-file-name>\`\n"
 
   exit 1
 fi
 
 ENVI="demo"
 DIRECTORY=$1
+ZIP_FILE_NAME=$2
```

Comment:
where is this used ?

Reply:
https://github.com/hmcts/bulk-scan-processor/blob/07253422fe21a0a9d3708f10ed408172c3cbe0aa/bin/sign-zip-upload.sh#L81 
and https://github.com/hmcts/bulk-scan-processor/blob/07253422fe21a0a9d3708f10ed408172c3cbe0aa/bin/sign-zip-upload.sh#L94